nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Ba 2 subscription

# Description
Added method in subscription repository and controller to get all active subscriptions by current user id (i.e. subscriber).  Also added scroll bars back into the app.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
pull down this branch and run it.   Ensure that all previously working features are still functional.  temporarily remove the authorize statement from the subscription controller.  open postman and send a get request to ``` https://localhost:5001/api/subscription/GetAll/2 ```. The final number is the userProfileId Number of the subscriber, change this number a few times and ensure that you get a list of subscriptions with that number as the ```subscriberUserProfileId```.  

Verify that there is a scroll bar.
# Checklist:
